pkg/scheduler/factory/factory.go

```diff
@@ -407,17 +406,8 @@ func (c *configFactory) invalidatePredicatesForPvUpdate(oldPV, newPV *v1.Persist
 		}
 	}
 	if utilfeature.DefaultFeatureGate.Enabled(features.VolumeScheduling) {
-		oldAffinity, err := v1helper.GetStorageNodeAffinityFromAnnotation(oldPV.Annotations)
-		if err != nil {
-			glog.Errorf("cannot get node affinity fo *v1.PersistentVolume: %v", oldPV)
-			return
-		}
-		newAffinity, err := v1helper.GetStorageNodeAffinityFromAnnotation(newPV.Annotations)
-		if err != nil {
-			glog.Errorf("cannot get node affinity fo *v1.PersistentVolume: %v", newPV)
-			return
-		}
-
+		oldAffinity := oldPV.Spec.NodeAffinity
```

Comment:
Technically this isn't needed because the beta NodeAffinity is immutable.  But if we decide in the future to make it mutable, then we won't forget to update this.

Reply:
Got it, I think I can remove this next pr.